packages/lesswrong/components/comments/CommentsItem/CommentsItemMeta.tsx

```diff
@@ -286,6 +298,14 @@ export const CommentsItemMeta = ({
         collectionName="Comments"
         hideKarma={post?.hideCommentKarma}
       />}
+      {nonSkipVotes?.nonSkipDoppelCommentVotes.length
+        ? <LWTooltip title={"How many times did this (real) comment pass the Turing Test?"}>
+          <div className={classes.doppelCommentVoteDisplay}>
+            {`${nonSkipVotes.doppelCommentVotesForComment!.length} / ${nonSkipVotes.nonSkipDoppelCommentVotes.length}`}
```

Comment:
you can probably remove the `!` if it's not asking for it for the other field (assuming this was something legacy)